src/app/loadout-builder/LoadoutBuilder.tsx

```diff
@@ -143,10 +144,10 @@ function mapStateToProps() {
   );
 
   return (state: RootState): StoreProps => {
-    const { loAssumeMasterwork } = settingsSelector(state);
+    const { loUpgradeSpendTier } = settingsSelector(state);
     return {
       statOrder: statOrderSelector(state),
-      assumeMasterwork: loAssumeMasterwork,
+      upgradeSpendTier: loUpgradeSpendTier,
```

Comment:
Is this worth validating?

Reply:
Should we convert it into the new enum automatically?

Reply:
Yup done both here and in the api. Note that I added it in two places until the real api types are in. This is because we can't import anything from initial-settings as the defaultLocale function uses local storage.